### PR TITLE
Add Windows Hello for Business Note

### PR DESCRIPTION
**Pull request by MK-Kamal**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/3602

https://docs.microsoft.com/en-us/windows/security/identity-protection/hello-for-business/hello-faq

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/security/identity-protection/virtual-smart-cards/virtual-smart-card-overview.md [hunk 1]
```diff
@@ -26,6 +26,9 @@ This topic for IT professional provides an overview of the virtual smart card te
 
 -   [Smart Cards](../smart-cards/smart-card-windows-smart-card-technical-reference.md)
 
+> [!NOTE]
+> [Windows Hello for Business](https://docs.microsoft.com/en-us/windows/security/identity-protection/hello-for-business/hello-identity-verification) is the modern, two-factor credential for Windows 10. Microsoft will be deprecating virtual smart cards in the future but not date at this time. Customers using Windows 10 and virtual smart cards should move to Windows Hello for Business. Microsoft will publish the date early to ensure customers have adequate lead time to move to Windows Hello for Business. Microsoft recommends new Windows 10 deployments to use Windows Hello for Business. Virtual smart card remain supported for Windows 7 and Windows 8.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/virtual-smart-cards/virtual-smart-card-overview.md]:
```suggestion
> [Windows Hello for Business](docs.microsoft.com/windows/security/identity-protection/hello-for-business/hello-identity-verification) is the modern, two-factor credential for Windows 10. Microsoft will be deprecating virtual smart cards in the future but no date has been set at this time. Customers using Windows 10 and virtual smart cards should move to Windows Hello for Business. Microsoft will publish the date early to ensure customers have adequate lead time to move to Windows Hello for Business. We recommend new Windows 10 deployments to use Windows Hello for Business. Virtual smart cards remain supported for Windows 7 and Windows 8.
```

Comment (MK-Kamal) [reply windows/security/identity-protection/virtual-smart-cards/virtual-smart-card-overview.md]: [!NOTE]
> [Windows Hello for Business](https://docs.microsoft.com/en-us/windows/security/identity-protection/hello-for-business/hello-identity-verification) is the modern, two-factor authentication  for Windows 10. Microsoft will be deprecating virtual smart cards in the future but not date at this time. Customers using Windows 10 and virtual smart cards should move to Windows Hello for Business. Microsoft will publish the date early to ensure customers have adequate lead time to move to Windows Hello for Business. Microsoft recommends new Windows 10 deployments to use Windows Hello for Business. Virtual smart card remain supported for Windows 7 and Windows 8.

**Comment by e0i** [comment]
@dulcemontemayor The PR that addresses issue #3602 is available for review. Thanks.

**Comment by e0i** [comment]
@DulceMontemayor Following up for updates.

**Comment by e0i** [comment]
@DulceMontemayor Following up for updates.
cc @Dansimp

File: windows/security/identity-protection/virtual-smart-cards/virtual-smart-card-overview.md [hunk 2]
```diff
@@ -26,6 +26,9 @@ This topic for IT professional provides an overview of the virtual smart card te
 
 -   [Smart Cards](../smart-cards/smart-card-windows-smart-card-technical-reference.md)
 
+> [!NOTE]
+> [Windows Hello for Business](docs.microsoft.com/windows/security/identity-protection/hello-for-business/hello-identity-verification) is the modern, two-factor authentication  for Windows 10. Microsoft will be deprecating virtual smart cards in the future, but no date has been set at this time. Customers using Windows 10 and virtual smart cards should move to Windows Hello for Business. Microsoft will publish the date early to ensure customers have adequate lead time to move to Windows Hello for Business. We recommend new Windows 10 deployments to use Windows Hello for Business. Virtual smart cards remain supported for Windows 7 and Windows 8.
```
Comment (nenonix) [inline windows/security/identity-protection/virtual-smart-cards/virtual-smart-card-overview.md]:
```suggestion
> [Windows Hello for Business]((../hello-for-business/hello-identity-verification.md)) is the modern, two-factor authentication for Windows 10. Microsoft will be deprecating virtual smart cards in the future, but no date has been set at this time. Customers using Windows 10 and virtual smart cards should move to Windows Hello for Business. Microsoft will publish the date early to ensure customers have adequate lead time to move to Windows Hello for Business. We recommend that new Windows 10 deployments use Windows Hello for Business. Virtual smart cards remain supported for Windows 7 and Windows 8.
```

**Comment by e0i** [comment]
The PR is being escalated.

**Comment by e0i** [comment]
@AndreaBarr The PR that has been approved by document author is ready to be merged. Thanks.

**Pull request merged**